pkg/tsdbctl/create.go

```diff
@@ -182,7 +182,10 @@ func (cc *createCommandeer) calculatePartitionAndChunkInterval(rateInHours int)
 }
 
 func rateToHours(sampleRate string) (int, error) {
-	parsingError := errors.New(`not a valid rate. Accepted pattern: [0-9]+/[hms]. Examples: 12/m`)
+	if sampleRate == "-1" {
+		return 0, errors.New(`sample rate not provided! Please provide sample rate with --rate flag in the format of [0-9]+/[hms]. Example: 12/m`)
```

Comment:
Better move this error one level up, or it will get parsed in a (false) parse error.